From 385d8ad770a12cd58c931f071794bf7d2b4b65f5 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Wed, 24 Mar 2010 13:54:07 -0500
Subject: [PATCH 151/505] ENGR00121953 linux-test: Cleanup scripts and
 makefile

Created MX53 string in test-utils.sh for use in SRTC script
Removed MX53 from Makefile in vpu test

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 test-utils.sh               | 1 +
 test/mxc_rtc/autorun-rtc.sh | 3 ++-
 test/mxc_vpu_test/Makefile  | 2 +-
 3 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/test-utils.sh b/test-utils.sh
index a9012cd..eb6b4b3 100755
--- a/test-utils.sh
+++ b/test-utils.sh
@@ -172,6 +172,7 @@ platform()
 		MX353StackBoard )	plat=IMX35_3STACK ;;
 		MX373StackBoard )	plat=IMX37_3STACK ;;
 		MX51* )       		plat=IMX51 ;;
+		MX53* )       		plat=IMX53 ;;
 
 		* )     ;;
 	esac
diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index 4e934ab..e9789a6 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -8,7 +8,8 @@ source /unit_tests/test-utils.sh
 STATUS=0
 
 echo "$(platform)"
-if [ "$(platform)" == IMX37_3STACK ] || [ "$(platform)" == IMX51 ]; then
+if [ "$(platform)" == IMX37_3STACK ] || [ "$(platform)" == IMX51 ] \
+    || [ "$(platform)" == IMX53 ]; then
 	rtc_test_param=--no-periodic
 	RTC_IRQS_EXPECTED=1
 else
diff --git a/test/mxc_vpu_test/Makefile b/test/mxc_vpu_test/Makefile
index 0f99cf7..50cbce6 100644
--- a/test/mxc_vpu_test/Makefile
+++ b/test/mxc_vpu_test/Makefile
@@ -11,7 +11,7 @@ OBJ = main.o \
       loopback.o
 
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX27ADS IMX31ADS IMX37_3STACK IMX51 IMX53
+INCLUDE_LIST:=IMX27ADS IMX31ADS IMX37_3STACK IMX51
 
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(INCLUDE_LIST)))
 TARGET = mxc_vpu_test.out
-- 
1.8.0

